traffic_mirroring.go

```diff
@@ -30,36 +25,36 @@ func main() {
 	}
 
 	packetSource := gopacket.NewPacketSource(handle, handle.LinkType())
-	max_size := 10
-	packets_list := make([][]byte,max_size) 
-	current_packet := 0
+	packetsList := make(chan []byte, 22)
```

Comment:
Shouldn't it be `[][]byte` (a slice of byte slice)? 

After testing if `[][]byte` works, could you test whether `[]*[]byte` (a slice of pointers to byte slice) could work? I hope it could decrease the memory footprint.